test/JsonApiDotNetCoreTests/IntegrationTests/QueryStrings/QueryStringTests.cs

```diff
@@ -63,6 +63,24 @@ public async Task Can_use_unknown_query_string_parameter()
         httpResponse.ShouldHaveStatusCode(HttpStatusCode.OK);
     }
 
+    [Theory]
+    [InlineData("")]
+    [InlineData("foo")]
+    public async Task Should_ignore_query_parameter_with_empty_name(string parameterValue)
```

Comment:
Looking at the names of existing tests in this file:
- Cannot_use_unknown_query_string_parameter
- Can_use_unknown_query_string_parameter
- Cannot_use_empty_query_string_parameter_value

I'd suggest:
```suggestion
    public async Task Can_use_empty_query_string_parameter_name(string parameterValue)
```